Makefile

```diff
@@ -222,21 +212,27 @@ TEST_TARGETS := test-unit test-unit-amino test-unit-proto test-ledger-mock test-
 # Test runs-specific rules. To add a new test target, just add
 # a new rule, customise ARGS or TEST_PACKAGES ad libitum, and
 # append the new rule to the TEST_TARGETS list.
-
 test-unit: ARGS=-tags='cgo ledger test_ledger_mock norace'
 test-unit-amino: ARGS=-tags='ledger test_ledger_mock test_amino norace'
 test-ledger: ARGS=-tags='cgo ledger norace'
 test-ledger-mock: ARGS=-tags='ledger test_ledger_mock norace'
 test-race: ARGS=-race -tags='cgo ledger test_ledger_mock'
 test-race: TEST_PACKAGES=$(PACKAGES_NOSIMULATION)
-
 $(TEST_TARGETS): run-tests
 
+# check-* compiles and collects tests without running them
+# note: go test -c doesn't support multiple packages yet (https://github.com/golang/go/issues/15513)
+CHECK_TEST_TARGETS := test-unit test-unit-amino
```

Comment:
That's the offendling line @marbar3778, this should be:

```makefile
CHECK_TEST_TARGETS := check-test-unit check-test-unit-amino
```

Reply:
Ah, right. Sorry didn't notice. Wrong copy-paste :face_with_head_bandage: